tests/update_images/test_azure.py

```diff
@@ -246,23 +248,19 @@ def test_get_all_images(mock_azure_image_versions, mock_azure_image_details):
 
 
 def test_format_image():
-    """Test transforming a single Azure image into a schema approved format."""
-    mocked_image = {
-        "architecture": "x86_64",
-        "hyperVGeneration": "v2",
-        "offer": "offer",
-        "publisher": "publisher",
-        "sku": "sku",
-        "urn": "publisher:offer:sku:7.6.2020082423",
-        "version": "7.6.2020082423",
```

Comment:
Any idea why the date is missing in the azure data dump we get from the cli tool? :thinking:

Reply:
Sorry misread date for data at first (it's friday) :p 
Which CLI tool are referring to?

Reply:
The mocked_image data above is what the format_image method expects (which itself is a product of two methods 'get_image_versions' and 'get_image_details'). But this raises a good point: I think the format_image methods should be private as it requires image data that is pre-processed. Additionally: Only AWS currently calls format_image everything else uses format_images. I think I'll create a  housekeeping task for this.

Reply:
Ah great! Thank you for your explanation! :)